Render links to external docs

fabsrc opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

### Description

Renders links to external docs in all locations where external docs can be used:

- Tag Object *(modified for consistency)*
- Schema Object
  - Object *(added*)
  - Array *(added)*
  - Primitive *(added)*
- OpenAPI Object / Swagger Object *(not changed)*
- Operation *(not changed)*

### Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->
<!--- Use the magic "Fixes #1234" format, so the issues are -->
<!--- automatically closed when this PR is merged. -->

Fixes #1545

### How Has This Been Tested?
<!--- Please describe in detail how you manually tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

*TODO*

### Screenshots (if appropriate):

<img width="517" alt="Bildschirmfoto 2021-10-15 um 18 18 03" src="https://user-images.githubusercontent.com/8853782/137521590-d4e3cc36-cf23-4120-a37a-c0513eaa12e4.png">
<img width="1251" alt="Bildschirmfoto 2021-10-15 um 18 18 33" src="https://user-images.githubusercontent.com/8853782/137521599-895aa5ff-5ffe-4d9b-b638-6cecbeda125d.png">
<img width="1217" alt="Bildschirmfoto 2021-10-15 um 18 20 10" src="https://user-images.githubusercontent.com/8853782/137521604-3318a4b7-1d4b-4723-b688-ff65b4ceb7db.png">


## Checklist
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

### My PR contains... 
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] No code changes (`src/` is unmodified: changes to documentation, CI, metadata, etc.)
- [ ] Dependency changes (any modification to dependencies in `package.json`)
- [ ] Bug fixes (non-breaking change which fixes an issue)
- [ ] Improvements (misc. changes to existing features)
- [x] Features (non-breaking change which adds functionality)

### My changes...
- [ ] are breaking changes to a public API (config options, System API, major UI change, etc).
- [ ] are breaking changes to a private API (Redux, component props, utility functions, etc.).
- [ ] are breaking changes to a developer API (npm script behavior changes, new dev system dependencies, etc).
- [x] are not breaking changes.

### Documentation
- [x] My changes do not require a change to the project documentation.
- [ ] My changes require a change to the project documentation.
- [ ] If yes to above: I have updated the documentation accordingly.

### Automated tests
- [ ] My changes can not or do not need to be tested.
- [x] My changes can and should be tested by unit and/or integration tests.
- [ ] If yes to above: I have added tests to cover my changes.
- [ ] If yes to above: I have taken care to cover edge cases in my tests.
- [ ] All new and existing tests passed.

tim-lai commented:
@fabsrc Thanks for the PR! First, to complete this PR, please add Cypress E2E tests to validate that `External docs` appears in the expected operations, and that the text doesn't appear where it shouldn't. Also, I am curious to know if you've tried some responsive testing, i.e. narrowing and widening of the viewport, and if there's any CSS impact.

fabsrc commented:
@tim-lai I updated the PR and added some Cypress E2E tests. Also checked the responsiveness, which seems to be fine. Could you please have another look?

tim-lai commented:
@fabsrc PR merged! Thanks for the contribution and followup!
